bin/init-aio.sh

```diff
@@ -22,30 +22,34 @@ while getopts d:h:r: option; do
   esac
 done
 
-export ACC=/config/morph_chain.gz
-
-/usr/bin/privnet-entrypoint.sh node --config-path /config --privnet &
-
-while [[ "$(curl -s -o /dev/null -w %{http_code} localhost:30333)" != "422" ]];
-do
-  sleep 2;
-done
-
 export NEOGO=/usr/bin/neo-go
 export WALLET=/config/node-wallet.json
 
-cd /config && ./bin/config.sh ContainerFee 0 && ./bin/config.sh ContainerAliasFee 0
-
 /usr/bin/neofs-ir --config /config/config-ir.yaml &
 
 while [[ -z "$(/usr/bin/neofs-cli control healthcheck --ir --endpoint localhost:16512 -c /config/cli-cfg-ir.yaml | grep 'Health status: READY')" ]];
 do
   sleep 2;
 done
 
+CONSADDR=$(jq -r .accounts[2].address "${WALLET}")
+SNADDR=$(jq -r  .accounts[0].address /config/wallet-sn.json)
+
+${NEOGO} wallet nep17 transfer \
+        --wallet-config /config/node-config.yaml \
+        -r http://localhost:30333 \
+        --from ${CONSADDR} --force \
+        --to ${SNADDR} \
+        --token GAS \
+        --amount 100
+
+sleep 2 # https://github.com/nspcc-dev/neo-go/issues/3244
```

Comment:
is `for` waiting for TX hard to implement via `neo-go` in bash?

Reply:
Really inconvenient this way.

Reply:
as a temp i mean. both ways ok to me